README.md

```diff
@@ -27,19 +27,14 @@ producing complex, multi-container applications.
 2. Get a Vagrantfile for your box. Users of the
 [Atomic Developer Bundle (ADB)](https://github.com/projectatomic/adb-atomic-developer-bundle) should download a [Vagrantfile from the repository](https://github.com/projectatomic/adb-atomic-developer-bundle/tree/master/components).
 
-3. Enable your desired service(s) in [Vagrantfile](Vagrantfile) as:
+3. Enable the plugin as:
 
-        config.servicemanager.services = 'openshift'
+        config.vm.provision = 'servicemanager'
```

Comment:
Does this mean one needs to explicitly set the service-manager as provisioner now? How come?